src/draive/conversation/realtime/__init__.py

```
@@ -0,0 +1,3 @@
+from draive.conversation.realtime.state import RealtimeConversation
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)